composer.json

```diff
@@ -30,7 +30,10 @@
         "mongodb/mongodb": "^1.1",
         "phpunit/phpunit": "^7.0",
         "predis/predis":   "~1.0",
-        "doctrine/coding-standard": "^6.0"
+        "doctrine/coding-standard": "^6.0",
+        "psr/cache": "^1.0",
+        "cache/integration-tests": "dev-master",
+        "symfony/phpunit-bridge": "^4.4 || ^5.0"
```

Comment:
Should we use this as an opportunity to use `"sort-packages": true`?

Reply:
We discussed about back then and prefer not to introduce phpunit-bridge. Testing for deprecations can be done with custom error handlers and we did in the ORM.

Reply:
> We discussed about back then and prefer not to introduce phpunit-bridge.

I agree. I currently need the bridge because the cache integration test that I‘ve used doesn‘t work without the bridge. My plan is to drop that dependency eventually.

Reply:
Aye, another +1 from me. I wasn't comfortable upgrading roave/doctrine-simple-cache to use this phpunit-bridge either, seems an unnecessary dependency, so assuming doctrine-simple-cache isn't assimilated by doctrine/cache, I may drop that dependency and just focus on our own tests. RE: https://github.com/php-cache/integration-tests/issues/105#issuecomment-545461444